### DISCUSSION TITLE
Expose a non-minified version for local development

### DISCUSSION THREAD

**Discussion by ghostling** [twilio-labs/paste#896] [Ideas] [answered]
There's been some occasions where I'd like to tweak something in Paste source to test my usage when developing locally. However, it seems that the source code exposed is minified. 

Not sure if there's a way Paste could expose it as a separate build artifact or just document an existing way I can accomplish this.

**Comment by TheSisb** [comment, marked as answer]
Hi @ghostling, thanks for the question.  I've made a ticket in our backlog to investigate this some more. My hunch is we could expose both `index.min.js` and a `index.js` in our builds.

**Reply by TheSisb** [reply]
😓 Forgot to circle back, but all Paste packages now ship with a dist/index.debug.js file.  This is the PR when the fix went in https://github.com/twilio-labs/paste/pull/1985
